CHANGES.txt

```diff
@@ -33,3 +33,10 @@ Bugfix
   MIMEAccept to behave more like the old version. See
   https://github.com/Pylons/webob/pull/356
 
+- ``acceptparse.AcceptValidHeader``, ``acceptparse.AcceptInvalidHeader``, and
+  ``acceptparse.AcceptNoHeader`` will now always ignore offers that do not
+  match the required media type ABNF when calling ``.acceptable_offers()``.
```

Comment:
First hit:
https://duckduckgo.com/?q=ABNF

abnf.net - Association of Black Nursing Faculty

Suggest a reference for the acronym ABNF:

```
:rfc:`5234`
```

which Sphinx will resolve to https://tools.ietf.org/html/rfc5234

Reply:
that's fine, fixed in the public api and changelog